### ISSUE TITLE
Create Wireframe for every defined role

### ISSUE THREAD

**Issue by codeDavidLopez** [amosproj/amos2022ss06-idunion-blockchain-dashboard#57]
## User story
1. As a user with a role
2. I want / need a Wireframe
3. So that the relevant information for the role is defined and designed

## Acceptance criteria
* A set of Wireframe for the different roles created
* Shows basic structure of the dashboard
* Shows which metrics to visualize
* Select one view to be implemented by spearate feature for each role

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## Feature DoD 
* Feature has been fully implemented
* All acceptance criteria were met
* All tests are passing
* No known errors
* Documentation updated
* Bill of materials updated

## Sprint Release DoD
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Peer code review passed
* Developers agreed to release
* Documentation updated
* Bill of materials updated

**Comment by codeDavidLopez** [comment]
uploaded wireframe for management view is somehow not complete - will be re-uploaded and fixed, but in general it is available.
